src/test/testdata/lsl/AVLTreeArkTsFunSignatures.lsl

```diff
@@ -0,0 +1,105 @@
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://gitee.com/openharmony/arkcompiler_runtime_core/blob/master/static_core/plugins/ets/stdlib/std/containers/AVLTree.ets";
```

Comment:
I strongly suggest removing any mention of third parties, especially trademarks and other IPs (intellectual properties).
```suggestion
    language "unknown"
    url "-";
```

Reply:
Fixed